### PR TITLE
fix(core/utils): Add non-normative classes

### PR DESCRIPTION
**Pull request by Johanna-hub**
As suggested in #1946 by Marcos, this creates a new const for non-normative classes for export

### CODE REVIEW

File: src/core/utils.js [hunk 1]
```diff
@@ -97,6 +97,10 @@ const fetchDestinations = new Set([
   "",
 ]);
 
+//these classes mean a section is non-normative
```
Comment (marcoscaceres) [inline src/core/utils.js]: As "section" is an element, maybe something like: 
```suggestion
 // CSS classes that indicate that a container element is non-normative.
```

File: src/core/utils.js [hunk 2]
```diff
@@ -97,6 +97,10 @@ const fetchDestinations = new Set([
...
+export const nonNormativeClasses =
```
Comment (marcoscaceres) [inline src/core/utils.js]: This being a CSS selector (and "classes" being ambiguous), I'd suggest: 

```suggestion
export const nonNormativeSelector =
```

File: src/core/utils.js (hunk 1 above)
Comment (saschanaz) [reply src/core/utils.js]: "a container element" somehow sounds like "a parent element" though.

Comment (marcoscaceres) [reply src/core/utils.js]: True, we can probably drop "container".

Comment (marcoscaceres) [reply src/core/utils.js]: Updated suggestion.

Comment (saschanaz) [reply src/core/utils.js]: How about:

```diff
- that indicate that
+ indicating
```

as IMO doubled "that" looks weird 😀

File: src/core/utils.js [hunk 3]
```diff
@@ -97,6 +97,10 @@ const fetchDestinations = new Set([
   "",
 ]);
 
+//CSS selector for matching elements that are non-normative
```
Comment (saschanaz) [inline src/core/utils.js]: nit: space after `//`

**Pull request merged**